python/strings_udf/strings_udf/_lib/cudf_jit_udf.pyx

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION.
```

Comment:
I would just do a thorough sweep at the end and update all the copyrights. In the past (or at least last year going from 21-22) we typically just used the date when the PR is merged, irrespective of when the work started.